ui/src/views/admin/settings/SettingsView.vue

```diff
@@ -1,82 +1,145 @@
+            <v-form
+              v-model="isFormValid"
+              class="admin-settings-form"
+              @keyup.native.enter="handleSubmit(onFormSubmit)"
+              @submit.prevent="handleSubmit(onFormSubmit)"
+            >
+              <v-container class="py-0">
+                <v-row>
+                  <v-col
+                    cols="12"
+                    class="my-0 py-0"
                   >
-                    Update Profile
-                  </v-btn>
-                </v-col>
-              </v-row>
-            </v-container>
-          </v-form>
-        </material-card>
+                    <ValidationProvider
+                        v-slot="{ errors }"
+                        name="E-mail"
+                        rules="required|email"
+                    >
+                      <v-text-field
+                        v-model.trim="formData.email"
+                        outlined
+                        required
+                        disabled
+                        :error-messages="errors[0]"
+                        label="E-mail"
+                        class="my-0 py-0"
+                      />
+                    </ValidationProvider>
+                  </v-col>
+                  <v-col
+                    cols="12"
+                    class="my-0 py-0"
+                  >
+                    <ValidationProvider
+                        v-slot="{ errors }"
+                        name="Password"
+                        rules="required|min:8"
+                    >
+                      <v-text-field
+                        v-model.trim="formData.password"
+                        outlined
+                        required
+                        hint="Minimum 8 characters"
+                        persistent-hint
+                        label="Password"
+                        class="my-0 py-0"
+                        :append-icon="isPasswordShown ? 'mdi-eye' : 'mdi-eye-off'"
+                        :type="isPasswordShown ? 'text' : 'password'"
+                        :error-messages="errors[0]"
+                        @click:append="isPasswordShown = !isPasswordShown"
+                      />
+                    </ValidationProvider>
+                  </v-col>
+                  <v-col
+                    cols="12"
+                  >
+                    <v-row justify="center">
+                      <v-btn
+                        color="primary"
+                        min-width="150"
+                        class="mt-2 mb-5"
+                        v-bind:disabled='!isDisabled'
+                        @click="handleSubmit(onFormSubmit)"
+                      >
+                        SAVE SETTINGS
+                      </v-btn>
+                    </v-row>
+                  </v-col>
+                </v-row>
+              </v-container>
+            </v-form>
+          </ValidationObserver>
+        </v-card>
+
+        <MaterialSnackbar
```

Comment:
This is good for now; however it should be replaced later. I have added a library so that we can show notification by calling one line of function; but it is still in my branch.